<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_editarproven</class>
 <widget class="QDialog" name="fm_editarproven">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Produto </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_editapro"/>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Preço Unitário</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_editapreco"/>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Quantidade</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_editaqtd"/>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="btn_confirmared">
     <property name="text">
      <string>Confirmar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancelaed">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
